PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx

```diff
@@ -531,13 +575,17 @@ struct FlowGFWOmegaXi {
           registry.fill(HIST("InvMassK0s"), v0.pt(), v0.mK0Short(), v0.eta(), cent);
           registry.fill(HIST("hEtaPhiVtxzPOIK0s"), v0.phi(), v0.eta(), vtxz);
           fGFW->Fill(v0.eta(), fV0PtAxis->FindBin(v0.pt()) - 1 + ((fK0sMass->FindBin(v0.mK0Short()) - 1) * nV0PtBins), v0.phi(), wacc * weff, 8);
+          if (cfgOutputNUAWeights)
+            fWeightsK0s->Fill(v0.phi(), v0.eta(), vtxz, v0.pt(), cent, 0);
         }
       } else if (PDGCode == kLambda0) {
         if (TMath::Abs(v0.mLambda() - 1.115683) < cfgv0_mlambdawindow) {
```

Comment:
Could you use the `MassLambda0` named constant  from `CommonConstants/PhysicsConstants.h` for the next iteration

Reply:
Thanks for your comment.